src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -222,6 +222,13 @@
 				</div>
 			</dl>
 		</NcAppSettingsSection>
+
+		<!-- Information about current version used. Talk Desktop has this in 'About' window -->
+		<p
+			v-if="!IS_DESKTOP"
```

Comment:
Does it work in a template directly without adding to setup? 🤔

Reply:
Fixed